bundles/org.openhab.binding.bluetooth/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -5,11 +5,32 @@
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
 	<channel-type id="rssi">
-		<item-type>Number</item-type>
+		<item-type>Number:Power</item-type>
```

Comment:
Are you sure? According to https://en.wikipedia.org/wiki/DBm, "dBm" is dimensionless.

Reply:
I just looked up the smarthome unit that used "dBm" as its symbol. `SmartHomeUnits.DECIBEL_MILLIWATTS` uses "Power" as its quantity type.

Also I think that the wiki article states that "dB" is dimensionless, not "dBm".

Reply:
We might have to fix SmartHomeUnits then...

> Also I think that the wiki article states that "dB" is dimensionless, not "dBm".

I guess you are just reading the wrong sentence in it. It states:
"The dBm is also dimensionless but since it compares to a fixed reference value the dBm rating is an absolute one."

Reply:
> "The dBm is also dimensionless but since it compares to a fixed reference value the dBm rating is an absolute one."

Huh, does this mean that I shouldn't be using `SmartHomeUnits.DECIBEL_MILLIWATTS` then since it is wrongly typed as a unit of Power? Should probably be fixed then.

I'll leave rssi as a regular Number type for now then until the units get sorted out. 
Sorry for the back and forth, I should have taken the time to read sentence following the one I posted.

Reply:
> Should probably be fixed then.

Wanna do a PR against openhab-core?

> I'll leave rssi as a regular Number type for now then until the units get sorted out.

Sounds like a reasonable solution for now.